editor/assets/stylesheets/main.scss

```diff
@@ -1,7 +1,10 @@
 @import '../../inserter/style';
 
-.gutenberg {
+body {
```

Comment:
mmm Right! I was assuming this CSS would be loaded only on this page. update to use the `body.toplevel_page_gutenberg`